### PR TITLE
Fixed UI styles for dark and light mode

### PR DESCRIPTION
**Pull request by kujtimprenkuSQA**

### CODE REVIEW

File: src/modal/component/Modal/Modal.styles.ts [hunk 1]
```diff
@@ -80,90 +84,90 @@ box-sizing: content-box;
...
-    padding: 0.55em 1.4em;
-    text-align: center;
-    color: inherit;
-    font-family: inherit;
-    transition: background 150ms ease-in-out;
-    line-height: 1.15;
-    cursor: pointer;
- }
+  background: inherit;
+  font-size: 14.2px;
```
Comment (lewis-sqa) [inline src/modal/component/Modal/Modal.styles.ts:108]: What's the reason for moving away from em?

File: src/modal/component/Modal/Modal.tsx [hunk 2]
```diff
@@ -274,8 +266,11 @@ function Modal(): JSX.Element {
                 What is a Wallet?
               </span>
               <div
-                className="info-description"
-                style={walletInfoVisible ? mountedStyle : unmountedStyle}
+                className={
+                  walletInfoVisible
+                    ? `info-description show-explanation`
+                    : `info-description hide-explanation`
+                }
```
Comment (lewis-sqa) [inline src/modal/component/Modal/Modal.tsx:273]: We could shorten this to:
```jsx
<div className={`info-description ${walletInfoVisible ? "show" : "hide"}-explanation`}>
```

File: src/modal/component/Modal/Modal.styles.ts (hunk 1 above)
Comment (kujtimprenkuSQA) [reply src/modal/component/Modal/Modal.styles.ts:108]: I moved to "px" from "em" because when we integrated the wallet-selector in the guest book example, the font size was affected and the UI would look different meaning in the guest book integration the font size  for buttons was too large and it did not look nice.

File: src/modal/component/Modal/Modal.tsx (hunk 2 above)
Comment (kujtimprenkuSQA) [reply src/modal/component/Modal/Modal.tsx:273]: Thank you, will improve it.

**Review by lewis-sqa — APPROVED** [review]
